heron/scheduler-core/src/java/org/apache/heron/scheduler/RuntimeManagerRunner.java

```diff
@@ -195,8 +195,11 @@ void updateTopologyHandler(String topologyName, Config updateConfig)
       throws TopologyRuntimeManagementException, PackingException, UpdateDryRunResponse {
     assert !potentialStaleExecutionData;
     String newParallelism = updateConfig.getStringValue(RUNTIME_MANAGER_COMPONENT_PARALLELISM_KEY);
+    LOG.info("newParallelism " + newParallelism);
```

Comment:
If these logs are for the testing purpose, remove them. 
If they will be kept, let's merge them into one line. `LOG.info(parallelise, container, runtimeConfig)`